src/pjpersist/interfaces.py

```diff
@@ -169,6 +187,21 @@ def isDirty():
     def setDirty():
         """Set dirty flag"""
 
+    def setTransactionOptions(
+        readonly: bool = None,
+        deferrable: bool = None,
+        isolation_level: int = None) -> None:
+        """Set the options for the future transaction
+
+        Options can only be set before the postgres transaction has started
+        """
+
+    def begin() -> None:
+        """Explicitly begin the transaction
+
+        If not called, transaction will start implicitly when first SQL
+         statement is executed.
+        """
```

Comment:
Is this not part of the `IDataManager` API?

Reply:
`begin` is not part of [IDataManager](https://transaction.readthedocs.io/en/latest/api.html#transaction.interfaces.IDataManager) FWICT. It became a bit of redundancy though, so maybe we can just remove it.

Reply:
Removed `begin` completely